Makefile

```diff
@@ -17,16 +24,42 @@ tidy:
 	@go mod tidy
 	@git diff --exit-code -- go.mod go.sum
 
-.PHONY: build
-build:
+build/$(ZEEKPATH).zip:
+	@mkdir -p build
+	@curl -L -o build/$(ZEEKPATH).zip \
+		https://github.com/brimdata/zeek/releases/download/$(ZEEKTAG)/zeek-$(ZEEKTAG).$$(go env GOOS)-$(ARCH).zip
+
+build/$(SURICATAPATH).zip:
+	@mkdir -p build
+	@curl -L -o build/$(SURICATAPATH).zip \
+		https://github.com/brimdata/build-suricata/releases/download/$(SURICATATAG)/suricata-$(SURICATATAG).$$(go env GOOS)-$(ARCH).zip
+
+build/dist/zeek: build/$(ZEEKPATH).zip
+	@mkdir -p dist
+	@unzip -q build/$(ZEEKPATH).zip -d build/dist
+	@touch build/dist/zeek
+
+build/dist/suricata: build/$(SURICATAPATH).zip
 	@mkdir -p dist
-	@go build -ldflags='$(LDFLAGS)' -o dist ./cmd/...
+	@unzip -q build/$(SURICATAPATH).zip -d build/dist
+	@touch build/dist/suricata
 
 .PHONY: zed
 zed:
 	@go mod download
 	@GOBIN="$(CURDIR)/bin" go install github.com/brimdata/zed/cmd/zq
 
+.PHONY: build
+build: build/dist/zeek build/dist/suricata
+	@go build -ldflags='$(LDFLAGS)' -o build/dist ./cmd/...
+
+.PHONY: release
+release: build
+	@cd build \
+		&& mv dist brimcap \
+		&& zip -r brimcap-$$(git describe --always --tags).$$(go env GOOS)-$$(go env GOARCH).zip brimcap \
```

Comment:
Why not use `$(VERSION)`here?

Reply:
According to the comment from the file I took this from in [brimdata/zeek](https://github.com/brimdata/zeek/blob/master/brim/release):

> # Can't use --diry with "git describe" on Windows because of the symlink
# shenanigans.

Reply:
That's specific to the jiggery-pokery earlier in [https://github.com/brimdata/zeek/blob/master/brim/release](https://github.com/brimdata/zeek/blob/master/brim/release). We don't do any of that here, so we can safely use `--dirty` (and `$(VERSION)`).

Reply:
Awesome!